Sources/Controllers/2.0/OATargetInfoViewController.mm

```diff
@@ -95,6 +96,7 @@ @implementation OATargetInfoViewController
     CGFloat _calculatedWidth;
     
     OARowInfo *_nearbyImagesRowInfo;
+    NSMutableArray <OAAbstractCard *> *_cards;
```

Comment:
Remove this variable. The cards should be stored in the collapsable view as before.